pages/tutorials/[slug].js

```diff
@@ -63,7 +63,7 @@ export default function Tutorial({ name, steps, id, slug }) {
 
       <ButtonContainer>
         {currentStep === 0 ? (
-          <Link href="/tutorials">
+          <Link href="/tutorials" passHref>
```

Comment:
I've added passHref here, although I'm not sure if that makes sense for a button.